app/views/overview.html

```diff
@@ -122,8 +122,8 @@ <h2>
                   <div class="component-label">Application</div>
                   <span ng-bind-html="app | highlightKeywords : overview.state.filterKeywords"></span>
                 </h2>
-                <div ng-if="route = overview.bestRouteByApp[app]" class="pull-right">
-                  <h3 class="overview-route">
+                <div class="overview-routes" ng-if="overview.routesToDisplayByApp[app].length > 0">
```

Comment:
We typically use the `size` filter since it works with both arrays and maps.

```html
<div class="overview-routes" ng-if="overview.routesToDisplayByApp[app] | size">
```